deepchecks/nlp/checks/data_integrity/conflicting_labels.py

```diff
@@ -46,17 +46,17 @@ class ConflictingLabels(SingleDatasetCheck, ConflictingLabelsAbstract):
     """
 
     def __init__(
-        self,
-        ignore_case: bool = True,
-        remove_punctuation: bool = True,
-        normalize_unicode: bool = True,
-        remove_stopwords: bool = True,
-        ignore_whitespace: bool = False,
-        n_to_show: int = 5,
-        n_samples: int = 10_000_000,
-        random_state: int = 42,
-        max_text_length_for_display: int = 30,
-        **kwargs
+            self,
```

Comment:
In general - missing tests for these. For instance, a check that makes sure that samples like 'a'*1000 aren't considered the same as 'a'*2000